packages/tabs/src/Tab/Tab.tsx

```diff
@@ -20,14 +20,16 @@ import { TabProps } from './Tab.types';
  * @param props.to Destination when name is rendered as `Link` tag.
  *
  */
-function Tab({ selected, children, ...rest }: TabProps) {
+function Tab({ children, _shouldRender, ...rest }: TabProps) {
```

Comment:
Instead of a private prop, it might be better to use context to store the value of `forceRender`. Then, you can read that value inside this component.

Reply:
@shaneeza I prefer not to reach for the context API in this case since it's only 1 nesting level deep. I also like keeping all the rendering/visibility logic in `TabPanel`

Reply:
I'm not convinced we should expose a prop we don't want consumers to use. What is the con of adding the rendering logic in this component?

Reply:
Agree that we probably don't want consumers to use the prop. I created a separate `TabsContext` to store that and the `selectedIndex`

For more context on why I was trying to avoid this initially. We have a weird existing pattern of taking a consumer defined `Tab` and breaking it into a `TabTitle` + `TabPanel`. Ideal future state: we have 2 separate components that the consumer defines separately. We are blocked on completely pursuing that due to the refactor burden it would place on consumers. I think it's better if there is less logic in `Tab` for that future state